dom/dom_c.php

```diff
@@ -1080,6 +1080,14 @@ public function count() {}
  */
 class DOMNamedNodeMap implements Traversable, Countable {
 
+    /**
+     * @var int
+     * The number of nodes in the map. The range of valid child node indices is 0 to length - 1 inclusive.
+     * @link https://www.php.net/manual/en/class.domnamednodemap.php#domnamednodemap.props.length
+     */
+    public $length;
+	
+	
```

Comment:
There's also tabs in there, it seems (difficult to tell on GH...).
```suggestion

```

Reply:
Thank you. I commited the suggestion. Feel free to squash it before mege, thank you :)